x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/upgraded_cluster/30_ml_jobs_crud.yml

```diff
@@ -77,27 +77,6 @@ setup:
         name: ".ml-state-write"
   - is_true: ''
 
----
```

Comment:
This test is not longer relevant as established model memory is tracked in a different way now